src/chunkGeometryCache.ts

```
@@ -0,0 +1,434 @@
+  }
+
+  /**
+   * Generate a hash for chunk block data
+   */
+  async generateBlockHash (blockStateIds: Uint16Array | number[]): Promise<string> {
+    const data = blockStateIds instanceof Uint16Array
+      ? blockStateIds
+      : new Uint16Array(blockStateIds)
+
+    const buffer = await crypto.subtle.digest('SHA-256', data.buffer)
+    const hashArray = Array.from(new Uint8Array(buffer))
```

ESLint

[error] Prefer the spread operator over `Array.from(…)`.

(unicorn/prefer-spread)